_includes/sponsors.html

```diff
@@ -0,0 +1,60 @@
+<section class="sponsors">
+	<h3>Sponsors</h3>
+  {% if site.data.sponsors.premier_partners %}
+  <div class="sponsor-section-wrapper">
+    <h4>Premier Partners</h4>
+    <ul class="sponsors-list premier-partner">
+      {% for sponsor in site.data.sponsors.premier_partners %}
+      <li>
+        <a href="{{ sponsor.link }}">
+          <img src="/2022/images/sponsors/{{sponsor.logo}}.svg" />
```

Comment:
```suggestion
          <img src="/images/sponsors/{{sponsor.logo}}.svg" />
```

let's not use images from 2022 in this year's page – that's confusing